Endpoint Service Tests (mocked) Failing

:clock11: **When did the problem start?**
5th of May

<!-- Indicate when the problem started -->

:camera: **Live badge**

<!-- Provide a link to the live badge in plain text and markdown.  -->
https://img.shields.io/endpoint?url=https%3A%2F%2Fshields.redsparr0w.com%2F2473%2Fmonday
![](https://img.shields.io/endpoint?url=https%3A%2F%2Fshields.redsparr0w.com%2F2473%2Fmonday)

:wrench: **Is the live badge working?**
Yes, just the mocked service tests that are failing
<!-- Indicate whether or not the live badge is working. -->

:link: **CircleCI link**
https://app.circleci.com/pipelines/github/badges/daily-tests/206/workflows/89b88e3f-f0c1-4599-ac83-2a068acbfcf2/jobs/1103
<!-- Provide a link to the failing test in CircleCI. -->

:beetle: **Stack trace**
There's 4 failing like this one
```
AssertionError: expected an element of [ 'max-age=500 s-maxage=500' ] to equal 'max-age=500'
    at Proxy.somethingAssert (node_modules/chai-things/lib/chai-things.js:148:37)
    at Proxy.assertEqual (node_modules/chai/lib/chai/core/assertions.js:1026:12)
    at Proxy.methodWrapper (node_modules/chai/lib/chai/utils/addMethod.js:57:25)
    at Proxy.somethingMethod (node_modules/chai-things/lib/chai-things.js:97:25)
    at Proxy.somethingMethod (node_modules/chai-things/lib/chai-things.js:122:33)
    at Proxy.overwritingMethodWrapper (node_modules/chai/lib/chai/utils/overwriteMethod.js:78:33)
    at Proxy.allMethod (node_modules/chai-things/lib/chai-things.js:165:25)
    at Proxy.overwritingMethodWrapper (node_modules/chai/lib/chai/utils/overwriteMethod.js:78:33)
    at /home/circleci/project/shields/node_modules/icedfrisby/lib/icedfrisby.js:570:38
    at IcedFrisbyNock._invokeExpects (node_modules/icedfrisby/lib/icedfrisby.js:1195:26)
    at IcedFrisbyNock.run (node_modules/icedfrisby/lib/icedfrisby.js:1260:12)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Context.<anonymous> (node_modules/icedfrisby/lib/icedfrisby.js:1348:9)
```

:bulb: **Possible solution**
Given that these are mocked tests, this was presumably caused by recently merged changes. Not sure if it's a simple matter of updating the test mocks/assertions or if a bug snuck in
<!--- Optional: only if you have suggestions on a fix/reason for the bug -->

<!-- Love Shields? Please consider donating $10 to sustain our activities:
👉  https://opencollective.com/shields -->

--- a/services/endpoint/endpoint.tester.js
+++ b/services/endpoint/endpoint.tester.js
@@ -207,7 +207,7 @@ t.create('cacheSeconds')
       cacheSeconds: 500,
     })
   )
-  .expectHeader('cache-control', 'max-age=500')
+  .expectHeader('cache-control', 'max-age=500 s-maxage=500')
 
 t.create('user can override service cacheSeconds')
   .get('.json?url=https://example.com/badge&cacheSeconds=1000')
@@ -219,7 +219,7 @@ t.create('user can override service cacheSeconds')
       cacheSeconds: 500,
     })
   )
-  .expectHeader('cache-control', 'max-age=1000')
+  .expectHeader('cache-control', 'max-age=1000 s-maxage=1000')
 
 t.create('user does not override longer service cacheSeconds')
   .get('.json?url=https://example.com/badge&cacheSeconds=450')
@@ -231,7 +231,7 @@ t.create('user does not override longer service cacheSeconds')
       cacheSeconds: 500,
     })
   )
-  .expectHeader('cache-control', 'max-age=500')
+  .expectHeader('cache-control', 'max-age=500 s-maxage=500')
 
 t.create('cacheSeconds does not override longer Shields default')
   .get('.json?url=https://example.com/badge')
@@ -243,7 +243,7 @@ t.create('cacheSeconds does not override longer Shields default')
       cacheSeconds: 10,
     })
   )
-  .expectHeader('cache-control', 'max-age=300')
+  .expectHeader('cache-control', 'max-age=300 s-maxage=300')
 
 t.create('Bad scheme')
   .get('.json?url=http://example.com/badge')

